packages/beacon-node/src/execution/engine/utils.ts

```diff
@@ -53,17 +50,58 @@ export function getExecutionEngineState({
 
     case ExecutePayloadStatus.UNAVAILABLE:
       return ExecutionEngineState.OFFLINE;
+
+    default:
+      // In case we can't determine the state, we assume it stays in old state
```

Comment:
this comment seems updated, wasn't the previous one more accurate?

```
// In case we can't determine the state, we assume it's online
// This assumption is better than considering offline, because the offline state may trigger some notifications
```